cli/medperf/utils.py

```diff
@@ -487,3 +474,21 @@ def get_stats(data_path: str, remove: bool = True) -> dict:
     if remove:
         os.remove(stats_path)
     return stats
+
+
+def get_results(result_path: str, remove: bool = True) -> dict:
+    """Retrieves the results of an execution
+
+    Args:
+        path (str): path to the result root
+
+    Returns:
+        dict: results of the execution.
+    """
+    path = os.path.join(result_path, config.results_filename)
+    with open(path, "r") as f:
+        results = yaml.safe_load(f)
+
+    if remove:
```

Comment:
What are the use-cases of `remove=True`?

I believe it'd be best to separate this into two functions. `get_results` removing results doesn't seem intuitive

Reply:
alright. yes, there is actually no rationale behind having this. will do that change for `get_stats` as well.